custom_components/moonraker/sensor.py

```diff
@@ -303,6 +304,59 @@ async def async_setup_optional_sensors(coordinator, entry, async_add_entities):
     async_add_entities([MoonrakerSensor(coordinator, entry, desc) for desc in sensors])
 
 
+async def async_setup_history_sensors(coordinator, entry, async_add_entities):
+    history = await coordinator.async_fetch_data(METHOD.SERVER_HISTORY_TOTALS)
+    if history.get("error"):
```

Comment:
I'm not sure how to test that condition. I'm a bit lost with the patch api and the returns value

Reply:
Have you tested what it throw when it fails?

Reply:
yes, it return {"error" : "Method not found"}